File: Uri 1012 - Área/Main.py
a,b,c= [float(x) for x in input().split()]
pi=3.14159
tri=(a*c)/2
circ=pi*c**2
trap=(a+b)*c/2
quad=b**2
reta=a*b
print('TRIANGULO: %.3f'%tri)
print('CIRCULO: %.3f'%circ)
print('TRAPEZIO: %.3f'%trap)
print('QUADRADO: %.3f'%quad)
print('RETANGULO: %.3f'%reta)

File: Uri 1012 - Área/Main.java
import java.util.Scanner;

public class Main {

public static void main(String[] args) {
        
        double a, b, c, terretan, areacir, trape, quadra, retan;
        Scanner in = new Scanner(System.in);
        a = in.nextDouble();
        b = in.nextDouble();
        c = in.nextDouble();
        terretan = (a * c)/ 2;
        areacir = 3.14159 * (Math.pow(c,2));
        trape = ((a + b) * c)/2;
        quadra = Math.pow(b,2);
        retan = (a * b) ;
        System.out.printf("TRIANGULO: %.3f%n", terretan);
        System.out.printf("CIRCULO: %.3f%n", areacir);
        System.out.printf("TRAPEZIO: %.3f%n", trape);
        System.out.printf("QUADRADO: %.3f%n", quadra);
        System.out.printf("RETANGULO: %.3f%n", retan);
        
        }           
}